<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DPMSSettings</class>
 <widget class="QWidget" name="DPMSSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DPMS Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>&amp;Enable display power management</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="standbyLabel">
        <property name="text">
         <string>Blank after</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_1">
        <item>
         <widget class="QSlider" name="standbySlider">
          <property name="maximum">
           <number>120</number>
          </property>
          <property name="tracking">
           <bool>true</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="standbyValueLabel">
          <property name="text">
           <string>Never</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="suspendLabel">
        <property name="text">
         <string>Put to sleep after</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QSlider" name="suspendSlider">
          <property name="maximum">
           <number>120</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="suspendValueLabel">
          <property name="text">
           <string>Never</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="offLabel">
        <property name="text">
         <string>Switch off after</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QSlider" name="offSlider">
          <property name="maximum">
           <number>120</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="offValueLabel">
          <property name="text">
           <string>Never</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
